### PR TITLE
travis: remove unnecessary database creation

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref: https://github.com/doctrine/dbal/commit/a66010640cc03e73a60cee16f1da36daabd63898

https://github.com/doctrine/doctrine2/pull/1251#discussion_r23018406

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Ping @deeky666 , sth like this?

File: tests/travis/mysql.travis.xml [hunk 1]
```diff
@@ -12,7 +12,6 @@
         <var name="tmpdb_host" value="localhost" />
         <var name="tmpdb_username" value="travis" />
         <var name="tmpdb_password" value="" />
-        <var name="tmpdb_name" value="doctrine_tests_tmp" />
```
Comment (stof) [inline tests/travis/mysql.travis.xml]: should it really be removed ?

Comment (deeky666) [inline tests/travis/mysql.travis.xml]: totally fine IMO as MySQL allows connecting without dbname. The temp connection is only for database creation AFAIK. Also travis is green :)

**Comment by deeky666** [comment]
@TomasVotruba hmm looks like it skips some tests now for now obvious reason. I guess it has to do with error silencing in the `TestUtils.php` maybe. We need to rework that anyways. Can you try to bake in the current [DBAL master version](https://github.com/doctrine/dbal/blob/master/tests/Doctrine/Tests/TestUtil.php)?

**Comment by deeky666** [comment]
Yeah it currently requires the `tmpdb_name` variable [here](https://github.com/doctrine/doctrine2/blob/master/tests/Doctrine/Tests/TestUtil.php#L38). So adopting the `TestUtils.php` should do the trick (was on my list anyways) and use the correct driver because otherwise it'll always run the tests with sqlite.

**Comment by TomasVotruba** [comment]
@deeky666 I'm not sure what exactly to rework. Just remove `$GLOBALS['tmpdb_name']`?

**Comment by Ocramius** [comment]
> ust remove `$GLOBALS['tmpdb_name']` ?

This seems wrong: instead, just provide a default value.

Tests using a tmp db should always run or eventually fail IMO.

**Comment by Ocramius** [comment]
Decided not to merge this for now: the performance overhead is negligible compared to the problems popping up in the test suite. Consider re-opening if this works with 

```xml
        <var name="tmpdb_name" value="doctrine_tests_tmp" />
```

**Comment by TomasVotruba** [comment]
@Ocramius Can you reopen? I've repushed just travis changes, but PR didn't update.

**Comment by Ocramius** [comment]
@TomasVotruba not reopenable for some reason: please make a new PR with that branch.

**Pull request closed without merging**